### ISSUE TITLE
unexpected closing

### ISSUE THREAD

**Issue by Grem96** [igorquintaes/ComicDaysDownloader#2]
Until several days ago it was working normally. Now it doesn't, I tried to download it again but the same problem appears. Is it due to some Windows 10 update?

**Comment by igorquintaes** [comment]
I don't think it could be... Maybe ComicDays made an update on their website and that changes break the way to obtain data to download. I can check it this night [gmt -3]

**Comment by igorquintaes** [comment]
@JovannyOviedo I am not able to reproduce, it is working fine in all my attempts. It would not to be like a Windows 10 update related problem, since it runs based on .NET Core 3.1 runtime. There is any log program on your Event Viewer? If not, I can develop a simple log writter soonly.

**Comment by Grem96** [comment]
![imagen](https://user-images.githubusercontent.com/28954326/93655395-3d05c400-f9e0-11ea-9cff-ea550e61f87f.png)

**Comment by igorquintaes** [comment]
Missing file error. This error should not be throwing, since Microsoft Visual Studio add it on .exe program. I will read about distributing and packing, since is not a code error. If you have some programming acknowledge, you can download all the source, compile and run. It should work.
